test/apps/iodemo/io_demo.cc

```diff
@@ -532,11 +540,15 @@ class P2pDemoCommon : public UcxContext {
                 memh = NULL;
             }
             return new Buffer(buffer, size, pool, memory_type, map_context,
-                              memh);
+                              memh, zcopy);
```

Comment:
`zcopy` is always `false` here - no need to pass it here

Reply:
done